productImport/mapping/handlerInterface.go

```diff
@@ -7,9 +7,10 @@ import (
 )
 
 type MappingHandlerInterface interface {
-	init(mappingConfigPath string) map[string]string
+	initProductsMapping() map[string]string
```

Comment:
Do you really need this method in interface?

Reply:
fixed